bubble_sort_test.rb

```diff
@@ -4,32 +4,12 @@
 require_relative './bubble_sort'
 
 class BubbleSortTest < Minitest::Test
-  def test_bubblesort_class
-    sorter = SortingSuite::BubbleSort.new
-    assert_equal "SortingSuite::BubbleSort", sorter.class.to_s
-  end
-
-  def test_bubblesort_sort_accepts_1_arg
-    sorter = SortingSuite::BubbleSort.new
-    assert_equal 1, sorter.method(:sort).arity
-  end
-
```

Comment:
These are good tests to have in order to get used to writing tests and doing TDD. After awhile, you'll notice that these tests stop providing a lot of value for the codebases you're working on, and it will make sense to stop doing them at that point.